I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/cable.rpy", line 25, in script
    if pasDeBras == True and jambe == True:
  File "game/cable.rpy", line 29, in <module>
    elif posture == True and pasDeBras == True:
NameError: name 'posture' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/cable.rpy", line 25, in script
    if pasDeBras == True and jambe == True:
  File "renpy/ast.py", line 1852, in execute
    if renpy.python.py_eval(condition):
  File "renpy/python.py", line 2249, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "renpy/python.py", line 2242, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/cable.rpy", line 29, in <module>
    elif posture == True and pasDeBras == True:
NameError: name 'posture' is not defined

Windows-10-10.0.18362
Ren'Py 7.4.2.1292
Robot et humain 1.0
Thu Feb 18 17:34:21 2021
